### ISSUE TITLE
New dice are temporarily shown above notch (onto external display)

### ISSUE THREAD

**Issue by ffried** [chockenberry/Notchmeister#24]
Given the same dual display setup as described in #23, the dice are shown on the upper display when hiding.
The video shows only the upper screen.

https://github.com/chockenberry/Notchmeister/assets/1689782/2c43811a-7ecd-46ff-a3fe-cf8da4417b44

**Comment by chockenberry** [comment]
Yeah, that's a hard problem to solve. The hiding effect moves the window off screen, but in your configuration that's not really off screen.

**Comment by ffried** [comment]
Just a random idea:
Maybe you could "collapse" the window at the top of the screen instead? If its view is no longer constraint to the top at that point, it should look the same, right?

**Comment by chockenberry** [comment]
That's a good idea and I just tried it. Unfortunately animating the window frame (to collapse) has issues clipping the animation layer content, so the dice just disappear suddenly.
